e2e/cloudfront/copilot/environments/test/manifest.yml

```diff
@@ -10,4 +10,5 @@ http:
         restrict_to:
           cdn: true
 
-cdn: true
\ No newline at end of file
+cdn:
+  terminate_tls: true
```

Comment:
i think this will fail validation without a cert...

Reply:
let me test...

Reply:
ok this is not expected at all. I'll work on a fix before merging this PR.